web/src/app/alerts/AlertsList.js

```diff
@@ -0,0 +1,369 @@
+import React, { useState } from 'react'
+import { PropTypes as p } from 'prop-types'
+import gql from 'graphql-tag'
+import {
+  Hidden,
+  Snackbar,
+  SnackbarContent,
+  Typography,
+  makeStyles,
+  isWidthDown,
+} from '@material-ui/core'
+import {
+  ArrowUpward as EscalateIcon,
+  Check as AcknowledgeIcon,
+  Close as CloseIcon,
+  Info as InfoIcon,
+} from '@material-ui/icons'
+import { useSelector } from 'react-redux'
+
+import AlertsListFilter from './components/AlertsListFilter'
+import AlertsListControls from './components/AlertsListControls'
+import CreateAlertFab from './CreateAlertFab'
+import UpdateAlertsSnackbar from './components/UpdateAlertsSnackbar'
+
+import { formatTimeSince } from '../util/timeFormat'
+import { urlParamSelector } from '../selectors'
+import { useMutation, useQuery } from '@apollo/react-hooks'
+import QueryList from '../lists/QueryList'
+import useWidth from '../util/useWidth'
+
+export const alertsListQuery = gql`
+  query alertsList($input: AlertSearchOptions) {
+    alerts(input: $input) {
+      nodes {
+        id
+        alertID
+        status
+        summary
+        details
+        createdAt
+        serviceID
+        service {
+          name
```

Comment:
We should ask for the `id` with `service {}` so that it joins/updates the correct document in cache, since we're fetching the name.
```suggestion
        service {
          id
          name
```